mobile/native/src/ln_dlc/mod.rs

```diff
@@ -1082,7 +1082,6 @@ fn ln_dlc_node_settings() -> LnDlcNodeSettings {
         off_chain_sync_interval: Duration::from_secs(5),
         on_chain_sync_interval: Duration::from_secs(300),
         fee_rate_sync_interval: Duration::from_secs(20),
-        dlc_manager_periodic_check_interval: Duration::from_secs(30),
```

Comment:
I think this is okay. The `periodic_check` is only looking at the blockchain and the oracle. The blockchain can only ever be updated after an on-chain sync anyway. We might miss something that the oracle posts for 5 minutes, but that's acceptable since we're already waiting 244 blocks to be able to spend the buffer transaction anyway.

Reply:
Isn't it 288 blocks?